frontend/scripts/react-components/static-content/static-content.actions.js

```diff
@@ -0,0 +1,32 @@
+import { NOT_FOUND, redirect } from 'redux-first-router';
+import camelCase from 'lodash/camelCase';
+
+export const STATIC_CONTENT__SET_MARKDOWN = 'STATIC_CONTENT__SET_MARKDOWN';
+
+export const getStaticContentFilename = location => camelCase(
+  `${location.type} ${(location.payload.section || '')}`
+);
+
+export const getStaticContent = () => (dispatch, getState) => {
+  const { location, staticContent } = getState();
+  const filename = getStaticContentFilename(location);
+  const notFound = location.type === NOT_FOUND;
+  if (
+    typeof staticContent.markdown[filename] === 'undefined' &&
```

Comment:
wouldn't `staticContent.markdown[filename] === undefined` be equivalent? just asking, no need to change if they're the same